foundation/ui/src/main/java/com/tapptitude/ui/theme/Color.kt

```diff
@@ -0,0 +1,67 @@
+package com.tapptitude.ui.theme
+import androidx.compose.ui.graphics.Color
+
+val md_theme_light_primary = Color(0xFF6750A4)
```

Comment:
These were generated using the [material theme builder](https://m3.material.io/theme-builder#/custom).